matchmaker/Assets/ServerObjects/RESTBaseSO.cs

```diff
@@ -0,0 +1,30 @@
+﻿using GPF.ServerObjects;
+using System.Collections.Generic;
+using System;
+
+namespace ServerObjects.Test
+{
+    public abstract class RESTBaseSO : ServerObject
+    {
+        protected static Dictionary<string, string> GetParamsFromQueryString(string queryString)
+        {
+            var myParams = new Dictionary<string, string>();
+            var pairParts = queryString.Substring(1).Split('&');
```

Comment:
I would consider regex here. See [this example](https://stackblitz.com/edit/js-8zsr8f?file=index.js) in Javascript.

I would also allow an option to force all param keys to lowercase. I would also perform URL decoding.